### PR TITLE
Add CNCF Landscape

### PR DESCRIPTION
**Pull request by czenker**

### CODE REVIEW

File: radar/2021-01-01/cncf-landscape.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title:      "CNCF Landscape"
```
Comment (czenker) [inline radar/2021-01-01/cncf-landscape.md:2]: I had a bit of trouble writing this article. Initially I understood we wanted to say that we want to trust the maturity levels of the applications listed in the CNCF landscape. But this does not really work: The Landscape is a big list of tools that might solve a certain problem. The **only** projects that have a maturity level assigned are [CNCF-supported projects](https://www.cncf.io/projects/).

So I think what we want to say is: We trust the evaluation of the CNCF with the maturity of their projects. But I suppose we trust vendors like Google (Kaniko) or HashiCorp (Vault) just the same way.

All in all I think the CNCF Landscape is definitely something for a link list (and could be mentioned in an appropriate Tech Radar item), but it is too specific to get its own entry.

**Comment by fbrnc** [comment]
After discussing this in the DevOps-Cop we decided to not mention the CNCF in our Techradar this year.

File: radar/2021-01-01/cncf-landscape.md (hunk 1 above)
Comment (danielpoe) [reply radar/2021-01-01/cncf-landscape.md:2]: @czenker   also ich finde es liest sich gut und weißt halt alle auf die liste und die arbeit die andere damit machen hin.

Würde es so übernehmen - es sei denn ihr habt noch Gründe es besser nicht zu tun?

**Comment by danielpoe** [comment]
Closed - will not be mentioned in this version

**Pull request closed without merging**